preprocessing/README.md

```diff
@@ -8,7 +8,9 @@ convert it into GeoJSON format.
 
 1. Download the latest raw market data and put in into the *raw*
 folder as *markets-berlin.json*.
-2. Convert the raw market data by running the following command from
+2. Manually reformat the downloaded JSON file using the [*Pretty JSON* plugin][pretty-json]
```

Comment:
Why do you first prettyfy it and then convert it?

Reply:
Good point. I explained my reasoning in the updated commit.